xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/attachmentslist.vm

```diff
@@ -19,10 +19,21 @@
 ## ---------------------------------------------------------------------------
 #if ("$!{request.forceTestRights}" == "1")#template("xwikivars.vm")#end
 #template('attachment_macros.vm')
-#set ($attachments = $doc.attachmentList)
-#sortAttachments($attachments $request.sortAttachmentsBy)
-#if ($attachments.size() > 0)
-  #displayAttachments($attachments)
-#else
-  <p class="noitems">$services.localization.render('core.viewers.attachments.noAttachments')</p>
-#end
+#set ($columns = ['mimeType', 'filename', 'version', 'filesize', 'date', 'author', '_actions'])
+#set ($columnsProperties = {
+  'mimeType': {'type': 'list', "html": true, 'class': 'XWiki.AllAttachments'},
```

Comment:
Depending on a wiki page is not nice. Does it at least degrade nicely if this page is missing?

Reply:
In case the page is missing you just don't have options to filter
![filter](https://user-images.githubusercontent.com/22794181/90111269-0e4c5c00-dd57-11ea-949c-d8f02e975dbd.png)